data_store/transformation/pathfinders/pf_transform_r1.py

```diff
@@ -24,7 +24,7 @@ def transform(df_dict: dict[str, pd.DataFrame], reporting_round: int) -> dict[st
         row["Local Authority"]: row["Reference"][:6] for _, row in project_details_df.iterrows()
     }
     project_name_to_id_mapping = {row["Full name"]: row["Reference"] for _, row in project_details_df.iterrows()}
-    transformed = {}
+    transformed: dict[int | str, pd.DataFrame] = {}
```

Comment:
question: Do we _actually_ have int keys here ever?

Reply:
Explained above the behaviour I observed.